extensions/EXT/EXT_multisampled_render_to_texture2.txt

```diff
@@ -0,0 +1,109 @@
+Name
+
+    EXT_multisampled_render_to_texture2
+
+Name Strings
+
+    GL_EXT_multisampled_render_to_texture2
+
+Contributors
+
+    Jeff Leger
+    Maurice Ribble
+    Krzysztof Kosinski
+    Craig Donner
+    Tobias Hector
+    Jan-Harald Fredriksen
+    Nigel Williams
+
+Contact
+
+    Jeff Leger (jleger 'at' qti.qualcomm.com)
+
+Status
+
+    Complete
+
+Version
+
+    Last Modified Date: February 27, 2017
+    Revision: 3
+
+Number
+
+    274
```

Comment:
Also, for future reference, should be "OpenGL ES Extension #275" not just "275" in the extension spec.